File: C#/UserInput.cs
using System;

namespace UserInput
{
    class Program
    {
        static void Main(string[] args)
        {
            Console.WriteLine("Whats your name? ");
            String name = Console.ReadLine();

            Console.WriteLine("Whats your age? ");
            int age = Convert.ToInt32(Console.ReadLine());

            Console.WriteLine("Hello " + name);
            Console.WriteLine("You are " + age + " years old ");
            Console.ReadKey();
        }
    }
}

File: C++/BroCode/UserInput.cpp
#include <iostream>

// cout << (insertion operator)
// cin >> (extraction operator)

int main() 
{
   std::string name;
   int age;

   std::cout << "What's your name?: ";
   std::cin >> name;

   std::cout << "What's your age?: ";
   std::cin >> age;

   std::cout << "Hello " << name << '\n';
   std::cout << "You are " << age << " years old";

   return 0;
}
